### ISSUE TITLE
Valid package names can't be parsed

### ISSUE THREAD

**Issue by dwalker-va** [uber/prototool#103]
**input:** 
```proto3
syntax = "proto3";

package group.v1;
```
**command:** `prototool lint test.proto`
**output:**
```
test.proto:3:14: found "." but expected [.proto element {comment|option|import|syntax|enum|service|package|message}]
```
**expected:** No output

I found https://github.com/emicklei/proto/issues/37 and this looks like the exact same issue. Does `github.com/emicklei/proto` need to updated?

Hoping to use this tool in my org's CI.

**Comment by AlexAPB** [comment]
I can repro locally. 
We're pinned to a hash that contains that fix: [emicklei/proto/#37](https://github.com/emicklei/proto/issues/37), However, emicklei explicitly prevents key works (ie `group`) being the first part of the package path, so `rpc.foo` and `oneof.bar` will all fail with an that same unhelpful error. 

I can't see anything in the [spec](https://developers.google.com/protocol-buffers/docs/reference/proto3-spec) that explicitly disallows this. @peter-edge what's your experience with this?

In any case, a fix would be needed in `emicklei/proto`

**Comment by dwalker-va** [comment]
@AlexAPB `group` is not a part of the proto3 spec but it is part of proto2 https://developers.google.com/protocol-buffers/docs/reference/proto2-spec

`emicklei/proto` might have to know whether its reading a proto2 or proto3 file to make the right call here if naming a package with a language keyword is really a naughty thing.

It seems I can set the syntax to proto2 on this and it still compiles via `prototool compile` though, so I don't know if it's a real restriction regardless of syntax version.

**Comment by peter-edge** [comment]
See https://github.com/emicklei/proto/issues/89.

**Comment by AlexAPB** [comment]
follow emicklei/proto#90 for the fix

**Comment by peter-edge** [comment]
This is fixed on v0.3.0.

**Comment by dwalker-va** [comment]
Thanks for the quick release! We were able to upgrade and immediately have this (questionably named) package being linted in our CI process.
